pkg/config/constant/api.go

```diff
@@ -85,7 +87,7 @@ const (
 	// EnvHome is the default env var parsed for determining the application home
 	EnvHome = "OKCTL_HOME"
 	// EnvClusterDeclaration specifies the path to the cluster declaration context
-	EnvClusterDeclaration = "CLUSTER_DECLARATION"
+	EnvClusterDeclaration = EnvPrefix + "_" + "CLUSTER_DECLARATION"
```

Comment:
Reason for change:
 * It should be simple to consume env vars, both internally and when importing okctl as a lib. Other okctl environment variable constants are exposed like this (see pkg/context/context.go), and I want importing environment variable keys to be consistent.
 * A future solution could look at using the Viper lib to consume env vars consistently